sklearn/feature_extraction/tests/test_text.py

```diff
@@ -892,6 +892,17 @@ def test_countvectorizer_vocab_dicts_when_pickling():
         assert_equal(cv.get_feature_names(), unpickled_cv.get_feature_names())
 
 
+def test_countvectorizer_string_object_as_input():
+    # Ensure that string object is not given as input.
+    cv = CountVectorizer()
+    exception = ValueError
+    test_str = "hello world!"
+
+    assert_raises(exception, cv.fit_transform, test_str)
```

Comment:
I'd rather see `ValueError` in place than use a variable.

Please use `assert_raise_message` (or `assert_raises_regexp`)